lms/envs/common.py

```diff
@@ -2690,11 +2690,11 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 
 ################################# CELERY ######################################
 
-CELERY_IMPORTS = (
+CELERY_IMPORTS = [
     # Since xblock-poll is not a Django app, and XBlocks don't get auto-imported
     # by celery workers, its tasks will not get auto-discovered:
     'poll.tasks',
-)
+]
 
```

Comment:
```suggestion

# Adds extra packages that doesn't get auto-imported (Example: XBlocks)
CELERY_EXTRA_IMPORTS = []
```

You say add something like this or add more formal documentation?

Already in celery imports there is a brief documentation:

```
# Since xblock-poll is not a Django app, and XBlocks don't get auto-imported
# by celery workers, its tasks will not get auto-discovered:
```

cc @robrap

Reply:
I meant more formal docs as [described in this how-to](https://code-annotations.readthedocs.io/en/latest/contrib/how_to/documenting_django_settings.html#example-documentation).

For example, something like:
```
# .. setting_name: CELERY_EXTRA_IMPORTS
# .. setting_default: []
# .. setting_description: Adds extra packages that don't get auto-imported (Example: XBlocks).
#    These packages are added in addition to those added by CELERY_IMPORTS.
```
You can see other settings documented like this in the same file.

These get extracted and documented here: https://edx.readthedocs.io/projects/edx-platform-technical/en/latest/

If you want, you are welcome to provide docs in this format for CELERY_IMPORTS as well, but that is not required.

Reply:
Thanks for the feedback @robrap :D It's ready